test/test_bc.py

```diff
@@ -210,3 +207,287 @@ def bnd_norm(vec):
+            print(f"{temper=}")
+
+            cv_interior_pairs = interior_trace_pairs(discr, uniform_state.cv)
+            cv_int_tpair = cv_interior_pairs[0]
+            state_pairs = make_fluid_state_trace_pairs(cv_interior_pairs, gas_model)
+            state_pair = state_pairs[0]
+            cv_flux_int = scalar_flux_interior(cv_int_tpair)
+            print(f"{cv_flux_int=}")
+
+            cv_flux_bc = wall.cv_gradient_flux(discr, btag=BTAG_ALL,
+                                               gas_model=gas_model,
+                                               state_minus=state_minus)
+            print(f"{cv_flux_bc=}")
+            cv_flux_bnd = cv_flux_bc + cv_flux_int
+
+            t_int_tpair = interior_trace_pair(discr, temper)
+            t_flux_int = scalar_flux_interior(t_int_tpair)
+            t_flux_bc = wall.temperature_gradient_flux(discr, btag=BTAG_ALL,
+                                                       gas_model=gas_model,
+                                                       state_minus=state_minus)
+            t_flux_bnd = t_flux_bc + t_flux_int
+
+            i_flux_bc = wall.inviscid_divergence_flux(discr, btag=BTAG_ALL,
+                                                      gas_model=gas_model,
+                                                      state_minus=state_minus)
+
+            nhat = thaw(discr.normal(state_pair.dd), actx)
+            bnd_flux = flux_func(state_pair, gas_model, nhat)
+            dd = state_pair.dd
+            dd_allfaces = dd.with_dtag("all_faces")
+            i_flux_int = discr.project(dd, dd_allfaces, bnd_flux)
+            bc_dd = as_dofdesc(BTAG_ALL)
+            i_flux_bc = discr.project(bc_dd, dd_allfaces, i_flux_bc)
+
+            i_flux_bnd = i_flux_bc + i_flux_int
+
+            print(f"{cv_flux_bnd=}")
+            print(f"{t_flux_bnd=}")
+            print(f"{i_flux_bnd=}")
+
+            from mirgecom.operators import grad_operator
+            dd_vol = as_dofdesc("vol")
+            dd_faces = as_dofdesc("all_faces")
+            grad_cv_minus = \
+                discr.project("vol", BTAG_ALL,
+                              grad_operator(discr, dd_vol, dd_faces,
+                                            uniform_state.cv, cv_flux_bnd))
+            grad_t_minus = discr.project("vol", BTAG_ALL,
+                                         grad_operator(discr, dd_vol, dd_faces,
+                                                       temper, t_flux_bnd))
+
+            print(f"{grad_cv_minus=}")
+            print(f"{grad_t_minus=}")
+
+            v_flux_bc = wall.viscous_divergence_flux(discr, btag=BTAG_ALL,
+                                                     gas_model=gas_model,
+                                                     state_minus=state_minus,
+                                                     grad_cv_minus=grad_cv_minus,
+                                                     grad_t_minus=grad_t_minus)
+            print(f"{v_flux_bc=}")
```

Comment:
Couldn't find my previous fix for these. Ugh.  I wedged some actual tests into the fray - just to make sure that the boundary functions are working properly.  We'll need more thorough tests for our new Mengaldo-based BCs, which should actually pass the tests we throw at it.

Reply:
Fixed these tests.   Incidentally, they were useful even the way they were because they'd still fail when stuff was mechanically broken in the boundary processing.   Regardless, should be fixed up now.